nexus/networking/src/firewall_rules.rs

```diff
@@ -0,0 +1,441 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Functionality related to firewall rules.
```

Comment:
Is the contents of this file basically just moved, not changed?

Reply:
Correct; only the function parameters changed (to replace `self` with `datastore`, etc.), and the corresponding changes to the function bodies.

Reply:
Yeah, I see that now.  Not a big deal either way.